samples/snippets/src/test/java/com/example/dialogflow/SetAgentIT.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.dialogflow;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/*
+ * We cannot test setAgent because Dialogflow ES can only have one agent 
+ * and if we create a agent it will delete the exisitng testing agent and 
+ * would cause all tests to fail
+ */
+
+public class SetAgentIT {
+  private static String PROJECT_ID = System.getenv().get("GOOGLE_CLOUD_PROJECT");
+
+  @Test
+  public void testCreateAgent()  {
+    try {
+      SetAgent.setAgent(PROJECT_ID, "");
+      Assert.assertTrue(false);
+    } catch (Exception e) {
+      Assert.assertTrue(true);
+    }
+  }
```

Comment:
This test should assert something from the return value or the capture std out and assert the contents of like https://github.com/googleapis/java-dialogflow/blob/master/samples/snippets/src/test/java/com/example/dialogflow/DetectIntentStreamIT.java

Reply:
i can't test setAgent because it will delete the existing testing agent, this test will fail thats why i'm using assertTrue because the setAgent will fail.